enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/Breakout.tsx

```diff
@@ -0,0 +1,136 @@
+import { useDisclosure } from "@mantine/hooks";
+import { useState } from "react";
+import { match } from "ts-pattern";
+
+import S from "embedding-sdk/components/private/InteractiveQuestion/components/Picker.module.css";
+import {
+  type ListItem as BreakoutListItem,
+  getBreakoutListItem,
+} from "metabase/query_builder/components/view/sidebars/SummarizeSidebar/BreakoutColumnList";
+import {
+  type UpdateQueryHookProps,
+  useBreakoutQueryHandlers,
+} from "metabase/query_builder/hooks";
+import { BreakoutPopover } from "metabase/querying/notebook/components/BreakoutStep";
+import { Button, Stack } from "metabase/ui";
+import * as Lib from "metabase-lib";
+
+import { MultiStepPopover } from "../../../util/MultiStepPopover";
+import { useInteractiveQuestionContext } from "../../context";
+import { BadgeList } from "../util/BadgeList";
+import { ToolbarButton } from "../util/ToolbarButton";
+type BreakoutItem = {
+  breakout?: Lib.BreakoutClause;
+  breakoutIndex?: number;
+};
+
+export const BreakoutInner = ({
+  query,
+  onQueryChange,
+  stageIndex,
+}: UpdateQueryHookProps) => {
+  const { onAddBreakout, onUpdateBreakout, onRemoveBreakout } =
+    useBreakoutQueryHandlers({ query, onQueryChange, stageIndex });
+
+  const breakouts = Lib.breakouts(query, stageIndex);
+
+  const items: BreakoutListItem[] = breakouts.map(breakout =>
+    getBreakoutListItem(query, stageIndex, breakout),
+  );
+
+  const [step, setStep] = useState<"picker" | "list">("list");
+
+  const [selectedBreakout, setSelectedBreakout] = useState<BreakoutItem>({
+    breakout: undefined,
+    breakoutIndex: undefined,
```

Comment:
nit: I think we can leave these out, as it would've already been clear from the `BreakoutItem` types